dev-guide/cluster-version-operator/dev/clusteroperator.md

```diff
@@ -91,38 +91,6 @@ Here are the guarantees components can get when they follow the rules we define:
 3. Prevent a user from clicking the upgrade button because components have one or more preflight criteria that are not met (e.g. nodes are at version 4.0 so the control plane can't be upgraded to 4.2 and break N-1 compat)
 4. Ensure other components are upgraded *after* your component (guarantee "happens before" in upgrades, such as kube-apiserver being updated before kube-controller-manager)
 
-### There are a set of guarantees components are expected to honor in return
```

Comment:
I prefer the strictness and clarity of the wording in this section.  Can we keep it and move the other changes here?

Reply:
Do you mean the title of the section or its content?
If it is the title, I will give it a try to keep it.
But if you mean the content, it would be hard because the source of the truth is the GoDocs from API.
It would be hard to maintain if they are different in the two places.

If you feel the wording in the API is not clear or strict enough, we can work things out there and then copy them here.
The current version of the API docs has been discussed and approved in the arch call recently and it has been reviewed by more audience.